boards/shields/g1120b0mipi/g1120b0mipi.overlay

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2023, NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/{
+	aliases {
+		kscan0 = &kscan_input_g1120b0mipi;
+	};
+
+	chosen {
+		zephyr,display = &rm67162_g1120b0mipi;
+		zephyr,keyboard-scan = &kscan_input_g1120b0mipi;
+	};
+
+	en_mipi_display_g1120b0mipi: enable-mipi-display {
+		compatible = "regulator-fixed";
+		regulator-name = "en_mipi_display";
+		enable-gpios = <&nxp_mipi_connector 32 GPIO_ACTIVE_HIGH>;
+		regulator-boot-on;
+	};
+};
+
+&nxp_mipi_i2c {
+	status = "okay";
+	ft3267@38 {
+		/*
+		 * Note- the actual controller present on this IC is a FT3267,
+		 * but the FT35336 driver in Zephyr supports this IC.
+		 */
+		compatible = "focaltech,ft5336";
```

Comment:
```suggestion
		compatible = "focaltech,ft3267";
```

and adjust driver to support multiple compatibles?

Reply:
Oh, just noticed a typo in the comment too: "FT35336"

Reply:
I agree, about the renaming being an issue- I don't want to mess up applications downstream here. Maybe we use the def-undef trick to cover the following compatibles:
- focaltech,ft5336
- focaltech,ft5xx6
- focaltech,ft6xx6
- focaltech,ft3267

I'd prefer to leave the Kconfig itself untouched, as that would also affect downstream applications

Reply:
I'm also siding for leaving this as is, counting on the ADI folks to come up with something more convenient down the road to handle driver families.

Reply:
It definitely can be fixed later, it's not a blocker to me, just noting this needs to be improved/fixed.